feature/gnmi/otg_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -173,6 +175,50 @@ func TestInterfaceCounters(t *testing.T) {
 	}
 }
 
+func validateInAndOutPktsPerSecond(t *testing.T, dut *ondatra.DUTDevice, i1, i2 *interfaces.InterfacePath) bool {
+	if deviations.InterfaceCountersFromContainer(dut) {
+		time.Sleep(10 * time.Second)
+		return true
+	}
+	inSamples := gnmi.Collect(t, dut.GNMIOpts().WithYGNMIOpts(ygnmi.WithSubscriptionMode(gpb.SubscriptionMode_SAMPLE), ygnmi.WithSampleInterval(1*time.Second)), i1.Counters().InUnicastPkts().State(), 10*time.Second)
+	outSamples := gnmi.Collect(t, dut.GNMIOpts().WithYGNMIOpts(ygnmi.WithSubscriptionMode(gpb.SubscriptionMode_SAMPLE), ygnmi.WithSampleInterval(1*time.Second)), i2.Counters().OutUnicastPkts().State(), 10*time.Second)
```

Comment:
Sample interval should be 30 seconds as this is the operational use case and pass/fail criteria.  

THere was some offline discussion of possibly a future version of this test could be a performance/scale test where we attempt to find the limit / fastest sample rate.  But that is considered out of scope for this functional test at this time.